docs/rst/knora-api-server/design-documentation/authentication.rst

```diff
@@ -26,146 +26,139 @@ Authentication in the Knora API Server
+To check the credentials, there is a special route called **/v1/authenticate**, which can be used to check if the
+credentials are valid.
 
-1. The login form on the client can use */v1/authentication* to check if
-   the email/password combination provided by the user is correct. The
-   email and password can be provided as URL parameters (see above).
+Usage Scenarios
+^^^^^^^^^^^^^^^
 
-2. on the server, this gets checked and a corresponding result as
-   described will be returned
+1. Create token by logging-in, send token on each subsequent request, and logout when finished.
+2. Send *email*/*password* credentials on every request.
 
-3. all subsequent calls can then send these credentials as
-   authentication header or URL parameters (email / password), and in
-   the case of a web client just the cookie.
 
-Step 1 and 2 are optional, and can be skipped, if prior checking of the
-credentials is not needed. Naturally, this won't work for a web client
-using cookies for authentication.
 
 Skipping Authentication
-^^^^^^^^^^^^^^^^^^^^^^^^^
+-----------------------
 
 There is the possibility to turn skipping authentication on and use a hardcoded
-user (Test User). In **application.conf** set the
-``skip-authentication = true`` and Test User will be always
-assumed.
+user (Test User). In **application.conf** set the ``skip-authentication = true``
+and Test User will be always assumed.
+
+
 
 Sipi (Media Server)
-^^^^^^^^^^^^^^^^^^^^^
-
-For authentication to work with the media server, we need to add support
-for cookies. At the moment the SALSAH-App would set BasicAuth heathers,
-but this only works for AJAX requests using ``SALSAH.ApiGet`` (``Put``, etc.).
-Since the medias are embedded as source tags, the browser would get them
-on his own, and doesn't know anything about the needed AuthHeathers.
-With cookies, the browser would send those automatically with every
-request. The media server can use the credentials of the user
-requesting something for accessing the RepresentationsRouteV1, i.e. make
-this request in the name of the user so to speak, then the
-RepresentationResponderV1 should have all the information it needs to
-filter the result based on the users permissions.
+-------------------
+
+For authentication to work with the media server, we need to add support for cookies. At the moment the SALSAH-App
+would set BasicAuth heathers, but this only works for AJAX requests using ``SALSAH.ApiGet`` (``Put``, etc.).
+Since the medias are embedded as source tags, the browser would get them on his own, and doesn't know anything about
+the needed AuthHeathers. With cookies, the browser would send those automatically with every request. The media server
+can use the credentials of the user requesting something for accessing the RepresentationsRouteV1, i.e. make this
+request in the name of the user so to speak, then the RepresentationResponderV1 should have all the information it
+needs to filter the result based on the users permissions.
```

Comment:
This is what Sipi does already, right?

Reply:
Yes, more or less. It's a bit out of date.